src/components/productDescription/children/aboutProduct/Images.jsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable react/prop-types */
+import { Box, Button, Image } from "@chakra-ui/react";
+import { CartButtons } from "./CartButtons";
+import { useEffect } from "react";
+
+const Images = ({ product, imageIndex, handleImageIndex }) => {
+  const descriptiveImages = product.DescriptiveImages;
+
+  useEffect(() => {
+    const handleKeyDown = (e) => {
+      if (e.key === "Tab") {
+        e.preventDefault();
+        handleImageIndex((prev) => (prev + 1) % descriptiveImages.length);
+      }
+    };
+
+    window.addEventListener("keydown", handleKeyDown);
+
+    return () => {
+      window.removeEventListener("keydown", handleKeyDown);
+    };
+  }, [descriptiveImages.length, handleImageIndex]);
```

Comment:
I don't think we need all this to set an image

i think button might have onFocus or on active callback/event